data/json/overmap/overmap_terrain/overmap_terrain_military.json

```
@@ -253,6 +253,24 @@
     "extras": "build",
     "flags": [ "RISK_HIGH", "SOURCE_WEAPON", "SOURCE_AMMO" ]
   },
+  {
+    "type": "overmap_terrain",
+    "id": "outpost_roof",
```

```suggestion
    [ "id": "outpost_roof", "outpost_cross_roof" ],
```

And remove duplicating definition of `outpost_cross_roof`.